src/test/scala/org/broadinstitute/dsde/workbench/sam/directory/PostgresDirectoryDAOSpec.scala

```diff
@@ -5,11 +5,11 @@ import org.broadinstitute.dsde.workbench.model.google.{GoogleProject, ServiceAcc
 import org.broadinstitute.dsde.workbench.sam.TestSupport
 import org.broadinstitute.dsde.workbench.sam.model.BasicWorkbenchGroup
 import org.postgresql.util.PSQLException
-import org.scalatest.{BeforeAndAfterEach, FlatSpec, Matchers}
+import org.scalatest.{BeforeAndAfterEach, FreeSpec, Matchers}
 
 import scala.concurrent.ExecutionContext.Implicits.global
 
-class PostgresDirectoryDAOSpec extends FlatSpec with Matchers with BeforeAndAfterEach {
+class PostgresDirectoryDAOSpec extends FreeSpec with Matchers with BeforeAndAfterEach {
```

Comment:
The main think to look at in this file is the conversion from FlatSpec to FreeSpec.